dps/utils/pandas_utilities.py

```diff
@@ -72,6 +72,22 @@ def divide_by_series(s1, s2):
 	s = s1.div(s2)
 	return s
 
+def add_dfs(l_df):
+	'''
```

Comment:
This is not actually used anywhere in the code. You are right that it will not add uncertainties together correctly. This is I think is one of the only draw backs of pandas, its inability to deal with value error tuple manipulations effectively. I can create a function which will handle this appropriately.